src/static/js/ace2_inner.js

```diff
@@ -449,10 +449,8 @@ function Ace2Inner() {
         inCallStackIfNecessary('setStyled', () => {
           fastIncorp(12);
           const clearStyles = [];
-          for (const k in STYLE_ATTRIBS) {
-            if (STYLE_ATTRIBS[k]) {
-              clearStyles.push([k, '']);
-            }
+          for (const k in Object.keys(STYLE_ATTRIBS)) {
```

Comment:
still errors for me in eslint?  Doesn't for you?

Reply:
Right. My fault.